Update discover-the-dashboard.md (#389)
akeneo/pim-helpcenter: content/md/first-steps/discover-the-dashboard.md

themes: first-steps
title: Discover the **dashboard** in details
popular: false
related: what-is-a-project, project-widget-dashboard, what-is-a-locale, imports, exports, review-products-proposals, access-rights-on-products
---

# Overview

> Akeneo's Dashboard allows you to easily follow up your catalog enrichment progress, review the collaborative projects advancement (number of products completed, in progress or to be done) and supervise your teammates activity with the Teamwork Assistant project's widget.   

Author: ```suggestion
Akeneo's Dashboard allows you to easily follow up your catalog enrichment progress, review the collaborative project's advancement (number of products completed, in progress or to be done) and supervise your teammates' activity with the Teamwork Assistant project's widget.   
```

> Akeneo's Dashboard allows you to easily follow up your catalog enrichment progress, review the collaborative projects advancement (number of products completed, in progress or to be done) and supervise your teammates activity with the Teamwork Assistant project's widget.   

Reviewer: ```suggestion
The Akeneo PIM Dashboard allows you to easily track catalog enrichment progress, review how projects are advancing -- showing the number of products completed, in progress or to be done -- and supervise your team's activity with the Teamwork Assistant project widget.   
```

> It will also enable you to overview the number of products complete and incomplete per channel and locale, check the last operations and access the jobs reports in one click.    

Reviewer: ```suggestion
It will also enable you to view the number of products that are complete and incomplete per channel and locale, check prior operations, and access the job reports in one click.    
```

> Finally, visualize the latest proposals for product changes you have to review. Akeneo's Dashboard displays you at a glance all the information you need to optimize your working day!    

Reviewer: ```suggestion
Finally, the dashboard shows the latest proposals for product changes that require your review. The Akeneo PIM Dashboard displays all the information you need at a glance to optimize your enrichment work!    
```

> Akeneo Dashboard is the default opening page, once you've logged in.

Reviewer: ```suggestion
Akeneo Dashboard is the default opening page once you've logged in.
```

## Teamwork assistant projects _(EE only)_

> In the Entreprise Edition, the first widget of the dashboard is the **widget gathering the projects created with the [Teamwork Assistant](what-is-a-project.html)**.

Reviewer: ```suggestion
In the Enterprise Edition, the first widget on the dashboard is the **widget gathering the projects created with the [Teamwork Assistant](what-is-a-project.html)**.
```
